tests/test/test_blocks.py

```diff
@@ -38,19 +38,47 @@ def setUp(self):
                 [
                     {
                         "type": "paragraph",
-                        "value": f'<p>This is a paragraph with a footnote. <footnote id="{uuid}">1</footnote></p>',
+                        "value": (
+                            f'<p>This is a paragraph with a footnote. <footnote id="{uuid}">[{uuid[:6]}]</footnote></p>'
```

Comment:
The `[{uuid[:6]}]` bit isn't strictly required since `FIND_FOOTNOTE_TAG` doesn't care what's between `<footnote></footnote>` but I opted to make it an accurate representation of what is actually stored in the database.